formulas/functions/text.js

```diff
@@ -284,19 +296,13 @@ const TextFunctions = {
     return TextFunctions.RIGHT(...params);
   },
 
-  SEARCH: (findText, withinText, startNum) => {
-    findText = H.accept(findText, Types.STRING);
-    withinText = H.accept(withinText, Types.STRING);
-    startNum = H.accept(startNum, Types.NUMBER, 1);
-    if (startNum < 1 || startNum > withinText.length) throw FormulaError.VALUE;
-
-    // transform to js regex expression
-    let findTextRegex = WildCard.isWildCard(findText)
-      ? WildCard.toRegex(findText, "i")
-      : findText;
-    const res = withinText.slice(startNum - 1).search(findTextRegex);
-    if (res === -1) throw FormulaError.VALUE;
-    return res + startNum;
+  SEARCH: (searchText, searchedInText, startingNum) => {
+    const findText = H.accept(searchText, Types.STRING);
+    const withinText = H.accept(searchedInText, null, undefined, false);
+    const startNum = H.accept(startingNum, Types.NUMBER, 1);
+    if(Array.isArray(withinText))
```

Comment:
Exercise for you, based on this comment, fix SEARCH
https://github.com/Lido-App/fast-formula-parser/pull/70/files#r786202074

Reply:
I changed this fyi, Git issues